packages/frontend/tests/acceptance/sidebar-test.ts

```diff
@@ -68,6 +68,29 @@ module('Acceptance | Sidebar', function(hooks) {
           assert.notOk(page.sidebar.contacts.offlineCount.isVisible, 'offline count is shown');
         });
 
+        module('pinned contact are to be shown', function(hooks) {
+          hooks.beforeEach(async function() {
+            const pinButton = document.getElementsByClassName('pin-button')[0];
```

Comment:
this should be a page-object reference. :-\

Reply:
Do you mean something like '[data-test-sidebar-contacts-header]'?

Reply:
naw, that's just a selector. I mean using https://discordapp.com/channels/619942056898592768/622495839595790347/641377625172082768